engines/wintermute/base/scriptables/script.cpp

```diff
@@ -618,7 +619,14 @@ bool ScScript::executeInstruction() {
 						_state = SCRIPT_WAITING_SCRIPT;
 						_waitScript->copyParameters(_stack);
 					}
-				} else {
+				}
+#ifdef ENABLE_FOXTAIL
+				else if (BaseEngine::instance().getGameId() == "foxtail" && !strcmp(methodName, "LoadItems")) {
+					// somehow FoxTail works just fine without LoadItems method
```

Comment:
How about logging a warning?

Reply:
Logged a warning in 39d4ceaafa256c1a311b040da1128894d9355d07